frontend/src/metabase/common/components/BrowserCrumbs/BrowserCrumbs.module.css

```diff
@@ -0,0 +1,22 @@
+.root {
```

Comment:
No changes, only emotion -> css. I originally wanted to fix wrapping here but decided not to do it right now